lib/netext/httpext/error_codes.go

```diff
@@ -233,7 +235,10 @@ func NewK6Error(code errCode, msg string, originalErr error) K6Error {
 
 // Error implements the `error` interface, so K6Errors are normal Go errors.
 func (k6Err K6Error) Error() string {
-	return k6Err.Message
+	if k6Err.OriginalError == nil {
+		return k6Err.Message
+	}
+	return fmt.Sprintf("%s: %s", k6Err.Message, k6Err.OriginalError)
```

Comment:
The [Windows failure](https://github.com/k6io/k6/pull/2045/checks?check_run_id=2711529014) seems to be because of this change.

Is this a wrong pattern after Go 1.13? To me it would make sense to render the original error as well, so I'm inclined to just update the [Windows error](https://github.com/k6io/k6/blob/2692aa9ce84cc9866953ce897b96767c089b4d6c/core/local/local_test.go#L1039).

Reply:
I think you shouldn't change this. Half of the purpose of `K6Error` is for us to be able to set a specific user-friendly error message, while at the same time keeping the original error accessible. If you want the original error message to be part of the new error message, you can always include it in the constructor, like you do in `ToURL`.

Reply:
OK, I think this is a bit clunky, but resolved by 283d4bdd.